build/buildorder.props

```diff
@@ -7,6 +7,5 @@
   </ItemDefinitionGroup>
 
   <ItemGroup>
```

Comment:
Can we just murder this file now or should we keep it in case we have a temporary need for buildorder building?

Reply:
Yes, we can murder this now, along with a lot of other code. See https://github.com/aspnet/AspNetCore/pull/6995